src/cf-atomic-component/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "cf-atomic-component",
+  "version": "1.4.0",
```

Comment:
Should this start below v1 or are we going to pick up where we left off when it was in it's own repo?

Reply:
I think picking up where we left off would be clearest, but happy to set it any way that makes sense.